lang/go/bindings/cne/packet_test.go

```diff
@@ -0,0 +1,125 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022-2023 Intel Corporation.
```

Comment:
Copyright date is not correct, needs to be `Copyright (c) 2023 Intel Corporation.`